### PR TITLE
rpm: use python3.4 on RHEL7 by default

### PR DESCRIPTION
**Pull request by tchaikov**
python3.4 is the native python3 before 7.6

Signed-off-by: Kefu Chai <kchai@redhat.com>

- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

### CODE REVIEW

**Comment by tchaikov** [comment]
https://shaman.ceph.com/builds/ceph/wip-rpm-python3-el7/cea9d18ced929a330f2299681ef93768449a05ed/

File: ceph.spec.in [hunk 1]
```diff
@@ -87,7 +87,13 @@
 
 %{!?_udevrulesdir: %global _udevrulesdir /lib/udev/rules.d}
 %{!?tmpfiles_create: %global tmpfiles_create systemd-tmpfiles --create}
+%if 0%{?rhel} == 7
+%{!?python3_pkgversion: %global python3_pkgversion 34}
+%{!?python3_version: %global python3_version 3.4}
```
Comment (tchaikov) [inline ceph.spec.in]: the "main" python3 in EPEL7 is now 3.6. see https://lists.fedoraproject.org/archives/list/epel-announce@lists.fedoraproject.org/message/EGUMKAIMPK2UD5VSHXM53BH2MBDGDWMO/ .

Comment (b-ranto) [reply ceph.spec.in]: I think building against 3.6 would be a step in the right direction given that rhel 7.7 might introduce python 3.6. Also, it should not matter what rhel minor version you use when you get the package from epel.

Comment (tchaikov) [reply ceph.spec.in]: @b-ranto do you mean we **always** use python3.6 as the default python3 for CentOS/RHEL7 build? or, probably we should add an option of `with_python3_other` to respect `python3_other_version`?

Comment (tchaikov) [reply ceph.spec.in]: @b-ranto #27417

**Pull request merged**